src/Controller/HomeController.php

```diff
@@ -2,21 +2,38 @@
 
 namespace App\Controller;
 
+use App\Entity\Figure;
+use App\Entity\Comment;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
-use App\Entity\Figure;
+use Doctrine\ORM\EntityManagerInterface;
 
 class HomeController extends AbstractController
 {
     /**
      * @Route("/", name="home")
      */
-    public function home()
+    public function home(EntityManagerInterface $entityManager)
     {
-        $repo = $this->getDoctrine()->getRepository(Figure::class);
-        $figure = $repo->findAll();
+        $figureList = $this->getDoctrine()->getRepository(Figure::class)->findAll();
```

Comment:
why is there two locations to delete a figure ?
You are now duplicating code. It would be better to have a proper route to handle the deletion of a figure